### PR TITLE
fix: add objectFit to video block

### PR DESCRIPTION
**Pull request by jianwei1**
# Description

Adding the objectFit field to VideoBlock as part of this [ticket](https://3.basecamp.com/3105655/buckets/29689605/todos/5529329941).

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Api only

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged into main

### CODE REVIEW

File: apps/api-journeys/src/app/modules/block/video/video.graphql [hunk 1]
```diff
@@ -81,6 +87,10 @@ type VideoBlock implements Block {
   action that should be performed when the video ends
   """
   action: Action
+  """
+  aspect ratio for VideoBlock
```
Comment (tataihono) [inline apps/api-journeys/src/app/modules/block/video/video.graphql:91]:
```suggestion
  how the video should display within the VideoBlock
```

File: apps/api-journeys/src/app/modules/block/video/video.graphql [hunk 2]
```diff
@@ -126,6 +136,10 @@ input VideoBlockCreateInput {
   True if the coverBlockId in a parent block should be set to this block's id.
   """
   isCover: Boolean
+  """
+  aspect ratio for VideoBlock
```
Comment (tataihono) [inline apps/api-journeys/src/app/modules/block/video/video.graphql:140]: same as above

File: apps/api-journeys/src/app/modules/block/video/video.graphql [hunk 3]
```diff
@@ -163,6 +177,10 @@ input VideoBlockUpdateInput {
   """
   posterBlockId: ID
   fullsize: Boolean
+  """
+  aspect ratio for VideoBlock
```
Comment (tataihono) [inline apps/api-journeys/src/app/modules/block/video/video.graphql:181]: same as above

File: apps/api-journeys/src/app/modules/block/video/video.graphql [hunk 4]
```diff
@@ -8,6 +8,12 @@ enum VideoBlockSource {
   youTube
 }
 
+enum VideoBlockObjectFit {
+  fill
+  fit
+  zoomed
```
Comment (tataihono) [inline apps/api-journeys/src/app/modules/block/video/video.graphql:14]: can we document each of these. See here for good descriptions of some of the props here: https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit

**Review by tataihono — CHANGES_REQUESTED** [review]

File: apps/api-journeys/src/app/modules/block/video/video.graphql [hunk 5]
```diff
@@ -7,6 +7,16 @@ enum VideoBlockSource {
...
 }
+"""
+fill = video is sized to fill the frame
+fit = the video is scaled to maintain it's aspect ratio, and still fit within the frame
+zoomed = fills entire frame while maintaining it's aspect ratio
+"""
+enum VideoBlockObjectFit {
+  fill
+  fit
+  zoomed
```
Comment (tataihono) [inline apps/api-journeys/src/app/modules/block/video/video.graphql:18]:
```suggestion
enum VideoBlockObjectFit {
	"""
	The video is scaled to maintain its aspect ratio while filling the entire VideoBlock. If the video's aspect ratio does not match the aspect ratio of the VideoBlock, then the video will be clipped to fit.
	"""
  fill
	"""
	The video is scaled to maintain its aspect ratio while fitting within the VideoBlock. The entire video is made to fill the VideoBlock, while preserving its aspect ratio, so the video will be "letterboxed" if its aspect ratio does not match the aspect ratio of the VideoBlock.
	"""
  fit
	"""
	12.5% of either side of the video is discarded (this has the effect of converting a 16:9 aspect ratio to 4:3). The remaining video is scaled to maintain its new aspect ratio while fitting within the VideoBlock.  The remaining video is made to fill the VideoBlock, while preserving its new aspect ratio, so the video will be "letterboxed" if its new aspect ratio does not match the aspect ratio of the VideoBlock. 
	"""
  zoomed
```

**Review by tataihono — CHANGES_REQUESTED** [review]
here is a more fuller comment copied mostly from mozilla.

**Comment by storyworks** [comment]
Hey can you hold off on updating this with latest main until we merge https://github.com/JesusFilm/core/pull/1073 ?

**Review by tataihono — APPROVED** [review]

**Pull request merged**